ReactSkia/textlayoutmanager/RSkTextLayoutManager.cpp

```diff
@@ -103,6 +129,9 @@ uint32_t RSkTextLayoutManager::buildParagraph (AttributedString attributedString
     ParagraphStyle paraStyle;
     auto fontSize = TextAttributes::defaultTextAttributes().fontSize;
     auto fontSizeMultiplier = TextAttributes::defaultTextAttributes().fontSizeMultiplier;
+    
+    int fontWeight = SkFontStyle::kNormal_Weight;
```

Comment:
It is SkFontStyle::Weight type right ?

Reply:
yes, 
In vscode, it is suggesting short form.

Reply:
Then better to use SkFontStyle::Weight.  In future if they change the data type in skia then this code will not be compatible.